src/main/java/com/nutomic/syncthingandroid/fragments/SettingsFragment.java

```diff
@@ -21,10 +24,14 @@
 
 import com.nutomic.syncthingandroid.R;
 import com.nutomic.syncthingandroid.activities.SyncthingActivity;
+import com.nutomic.syncthingandroid.preferences.WifiSsidPreference;
 import com.nutomic.syncthingandroid.syncthing.RestApi;
 import com.nutomic.syncthingandroid.syncthing.SyncthingService;
 
+import java.util.HashSet;
 import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
```

Comment:
Ah I didn't think it was on purpose. The performance difference should be really minor, so do it however you want.